app/components/forms/events/view/create-access-code.js

```diff
@@ -0,0 +1,90 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+
+const { Component, computed } = Ember;
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+
+      fields: {
+        accessCode: {
+          identifier : 'access_code',
+          rules      : [
```

Comment:
Also, add a rule to accept only alphanumeric characters and hyphen/underscore. (`a-z,A-Z,0-9,-,_`)